setup.sh

```diff
@@ -115,6 +115,12 @@ then
   fi
 fi
 
+MODEL_DIR="${MODEL_DIR}"/"${MODEL}-${NUM_GPUS}gpu"
+CONF_PATH="/model/fastertransformer/config.pbtxt"
```

Comment:
CONF_PATH?? What for?

Reply:
see here: https://github.com/moyix/fastertransformer_backend/pull/1/commits/24ec694281bbceb666144274e51bb0fb55535977
tune.py reads off 'config.pbtx' from this env var. It can be changed to some other 'config.pbtx' other than this default if needed.